state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -962,6 +993,31 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Apply a list of configuration updates to the pallet.
+		///
+		/// Requires Governance.
+		#[pallet::call_index(11)]
+		#[pallet::weight(<T as frame_system::Config>::SystemWeightInfo::set_storage(updates.len() as u32))]
+		pub fn update_pallet_config(
+			origin: OriginFor<T>,
+			updates: BoundedVec<PalletConfigUpdate, ConstU32<10>>,
```

Comment:
As discussed, I don't think there is a huge disadvantage to slightly overestimating the number of assets here. I think we want to change it to support, say 100 elements, so we are able to update all assets in a single extrinsic in the forseeable future.